python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -958,7 +961,8 @@ def loadPixelBox(self, bbox, wcs, filterName=None, photoCalib=None, epoch=None):
         circle = self._calculateCircle(bbox, wcs)
 
         # find objects in circle
-        self.log.info("Loading reference objects using center %s and radius %s deg",
+        self.log.info("Loading reference objects from %s using center %s and radius %s deg",
+                      self.config.ref_dataset_name,
```

Comment:
Does this need to have a new line at the end here? I think the args will still fit and be within 110 characters.